src/Cake.Common/Tools/SignTool/SignToolSignSettings.cs

```diff
@@ -33,5 +33,10 @@ public sealed class SignToolSignSettings : ToolSettings
         /// Gets or sets the signed content's description.
         /// </summary>
         public string Description { get; set; }
+
+        /// <summary>
+        /// Gets or sets the signed content's expanded description URL.
+        /// </summary>
+        public Uri DescriptionUri { get; set; }
```

Comment:
@dbertram Gotcha!  I definitely prefer consistency over anything else.  For now, I would suggest that you leave it as Uri, and we need to ensure consistency across the various Settings classes within the Cake.Common project.  For example here:

https://github.com/cake-build/cake/blob/develop/src/Cake.Common/Tools/GitLink/GitLinkSettings.cs#L14